packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -378,9 +378,6 @@ $side-padding: 1.2em;
         }
     }
 
-    &__filter {
-        justify-content: flex-end;
-    }
```

Comment:
beside of this we have another role that say `@media (min-width: 1024px) { .reports__meta {align-items: center;} }` and on the other hand this part of application would be fixed width as screen size increase. as a result, I don't think we need this part. if we remove the above code, the issue will fix